autopilot/contractor.go

```diff
@@ -51,7 +54,9 @@ func newContractor(ap *Autopilot) *contractor {
 	}
 }
 
-func (c *contractor) applyConsensusState(cfg Config, state bus.ConsensusState) {
+func (c *contractor) ApplyConsensusState(cfg Config, state bus.ConsensusState) {
```

Comment:
I exported some methods to make it clearer that they are not called by the contractor on itself but from outside.